test/ui-testing/CustomInsert.test.js

```diff
@@ -1,4 +1,4 @@
-const timeout = process.env.SLOWMO ? 60000 : 10000;
+const timeout = process.env.SLOWMO ? 60000 : 60000;
```

Comment:
Could we remove this? 
```suggestion
const timeout = process.env.SLOWMO ? 60000 : 10000;
```

Reply:
Oh, yes, sorry. I forgot that.
And I also hope it didn't contribute to making the test successful 🤣